Bugfix sorter

leegosx opened the pull request:
Змінив повністю логіку, тому прошу уважно перевірити.  Дякую)

leegosx committed:
update and bugfix for sorter

new update and bugfix for sorter

leegosx committed:
Revert "update and bugfix for sorter"

This reverts commit 76e1b59e65b2f218ef9bf5d587d90ace405655b9.

leegosx committed:
bugfix for sorter

new method soultion

UkrainianEagleOwl commented on src/base_team_project/sorter.py:
```diff
@@ -1,83 +1,92 @@
-            exts = folder_names[folder]
-            if any(file_path.suffix[1:] in exts for file_path in file_paths):
-                folder_path.mkdir()
+    TRANS = {}
```
Було б добре якщо заповнення і створення TRANS винести окремо. Окрема функція яка буде заповнювати глобальну змінну Транс. Щоб робити це один раз при запуску програми, а не кожного разу при виклику сортувальника.

UkrainianEagleOwl commented on src/base_team_project/sorter.py:
```diff
@@ -1,83 +1,92 @@
+    unpack_archives(path)
+    sort_files(path)
 
-def sort_files_in_this_path(input_path):
```
Поверни виклик через функцію який я робив. Він потрібен для того щоб його викликати з команди у основній програмі, там для цього все було настроено, якщо основний виклик змінився потрібно просто його наладити.

leegosx committed:
Update sorter.py

я змінив словник TRANS, він тепер глобальний, та додав гарний вивід після успішного сортування файлів

leegosx committed:
update sorter
